package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/lifion/lifion-kinesis#readme",
   "main": "lib/index.js",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=17.0.1 || ^16 <16.10.0 || <16 >=10.0.0"
```

Comment:
please simplify, left should be lower than the right (easier to understand and read)

Reply:
don't use "16" "14", but rather the versions when things changed (look at node.green)